demo/scripts/controlsV2/sidePane/presets/PresetPlugin.ts

```diff
@@ -0,0 +1,38 @@
+import PresetPane, { PresetPaneProps, PresetPaneState } from './PresetPane';
+import { IEditor, PluginEvent } from 'roosterjs-content-model-types';
+import { SidePaneElementProps } from '../SidePaneElement';
+import { SidePanePluginImpl } from '../SidePanePluginImpl';
+
+export class PresetPlugin extends SidePanePluginImpl<PresetPane, PresetPaneProps> {
+    constructor() {
+        super(PresetPane, 'Preset', 'Presets');
+    }
+
+    initialize(editor: IEditor): void {
+        this.editor = editor;
+    }
+
+    onPluginEvent(event: PluginEvent) {
+        switch (event.eventType) {
+            case 'editorReady':
+            case 'contentChanged':
+                this.updatePresetPluginState();
```

Comment:
No need to update editor for ContentChanged event. Just do it for EditorReady should be ok.

Reply:
It needs to do it to update the CSS of the selected Preset button.